### DISCUSSION TITLE
Ring Spotlight Cam dual battery status

### DISCUSSION THREAD

**Discussion by dougmeek** [tsightler/ring-mqtt#488] [Support Q&A]
Firstly, thanks for the work you've done on this project, @tsightler. You've made my HA journey far more productive and functional.

With that said, I have four battery powered Ring Spotlight Cams that I use for surveillance on the exterior of the house. They each have two batteries. When looking at their battery level passed by ring-mqtt to HA, I only see the level of one battery. With the basic Ring integration, these would show up as fully charged at 200%. With one battery depleted and the other full, the battery level will show depleted for the entire device.

For example: I currently have a camera with one battery at 94% and the other battery at 6%. In the HA Ring integration and Ring app/web, this shows 100%, and with ring-mqtt it shows 6%.

Before I open an issue or feature request, I wanted to reach out and check to see if this is expected behavior or if there's something wrong with my implementation. See the screenshots attached for what I'm seeing.

Thanks again!

![image](https://user-images.githubusercontent.com/30700062/198424221-9ab39cb1-57cc-4598-bff5-7da676b4edc8.png)

![image](https://user-images.githubusercontent.com/30700062/198424278-2c8a09d9-aecb-4cd7-8604-a270c01f65a5.png)

**Comment by tsightler** [comment]
There's actually already a closed issue that talks about this a little.  Currently, the code simply reports the value returned in the battery_percentage property of the device health API.  I don't know what logic Ring uses to calculate this value, but I had assumed it would be the value of the currently active or lowest charged battery, but I don't have an idea for sure.

Of course it would be possible to improve this, but I don't like the idea of reporting more than 100% because, while Home Assistant seems OK with this, many systems are unable to deal with battery levels greater than 100% and, unlike the native Home Assistant component, ring-mqtt is not Home Assistant specific.

The ideal thing to do would probably be to report thee left/right battery as separate attribute values for those that want to monitor it more specifically but, as I don't even own any battery powered Ring cameras it's not clear to me how the API really behaves with two batteries.  I had assumed reporting battery_percentage from device health API would have been logical and would return a reasonable battery level that represents the overall device state.

I'm certainly open to accepting a PR that improves this, or, if someone wants to share some cameras with me for a few weeks, I'm willing to learn enough about the API to improve the logic myself.
